modules/administration/pages/custom-channels.adoc

```diff
@@ -0,0 +1,78 @@
+
+This section gives more detail on how to create, administer, and delete custom channels.
+
+
+
+== Creating Custom Channels
+
+If you have custom software packages that you need to install on your {productname} systems, you can create a custom child channel to manage them.
+You will need to create the channel in the {productname} {webui} and create a repository for the packages, before assigning the channel to your systems.
+
+You can select a vendor channel as the base channel if you want to use packages provided by a vendor.
+Alternatively, select ``none`` to make your custom channel a base channel.
+
+If you include a package with [systemitem]``-release`` at the end of the package name in your channel, it will be automatically installed when the channel is installed.
+
+[IMPORTANT]
+====
+Do not create child channels containing packages that are not compatible with the client system.
+====
+
+
+
+.Procedure: Create a custom channel
+
+. In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click btn:[Create Channel].
+. On the [guimenu]``Create Software Channel`` page, give your channel a name (for example, [systemitem]``My Tools SLES 15 SP1 x86_64``) and a label (for example, [systemitem]``my-tools-sles15sp1-x86_64``).
+Note that labels cannot contain space characters or capital letters.
+. In the [guimenu]``Parent Channel`` drop down, choose the relevant base channel (for example [systemitem]``SLE-Product-SLES15-SP1-Pool for x86_64``).
+Ensure that you choose the compatible parent channel for your packages.
+. In the [guimenu]``Architecture`` drop down, choose the appropriate hardware architecture (for example, [systemitem]``x86_64``).
+. Provide any additional information in the contact details, channel access control, and GPG fields, as required for your environment.
+. Click btn:[Create Channel]
+
+.Procedure: Create a software repository
+
+. In the {productname} {webui}, navigate to menu:Software[Manage > Repositories], and click btn:[Create Repository].
+. On the [guimenu]``Create Repository`` page, give your repository a label (for example, [systemitem]``my-tools-sles15sp1-x86_64-repo``).
+. In the [guimenu]``Repository URL`` field, provide the path to the folder that contains the [path]``repodata`` file (for example, [systemitem]``file:///opt/mytools/``).
+You can use any valid addressing protocol in this field.
+. Uncheck the [guimenu]``Has Signed Metadata?`` check box.
+. Provide any additional information in the SSL fields, as required for your environment.
+. Click btn:[Create Repository]
+. Synchronize your new repository with your custom channel by navigating to menu:Software[Manage > Channels], clicking on the name of your newly created custom channel, and navigating to the [guimenu]``Repositories`` tab.
+. Ensure the repository you want to synchronize is checked, and click btn:[Update Repositories].
```

Comment:
By checking the checkbox, you assign the repository to the channel. This selection is permanent until you change it.
Clicking the 'Sync' button in a one time action.

Not sure if the assignment of repos to channel needs to be expressed more prominent?

Reply:
Split this to its own procedure and emphasised that it's about assigning, not synchronising.